easybuild/easyconfigs/v/VASP/VASP-5.4.1-intel-2016.02-GCC-4.9.eb

```diff
@@ -12,17 +12,34 @@ toolchain = {'name': 'intel', 'version': '2016.02-GCC-4.9'}
 toolchainopts = {'usempi': True}
 
 # Vasp is proprietary software, see http://www.vasp.at/index.php/faqs on how to get access to the code
-sources = [SOURCELOWER_TAR_GZ]
+sources = ['vasp.5.4.1.24Jun15.tar.gz']
```

Comment:
@boegel `vasp.5.4.1.24Jun15.tar.gz` is the original filename (in VASP's download portal) of the `5.4.1` version. Do you mean add a new easyconfig for `vasp.5.4.1.05Feb16.tar.gz` (the one with GPU support)? I can add but we're not using GPUs so cannot test.